ffi/blitbuffer.lua

```diff
@@ -658,6 +647,18 @@ end
 function BB_mt.__index:getType()
     return rshift(band(MASK_TYPE, self.config), SHIFT_TYPE)
 end
+
+-- Determine if a pair of buffers can use CBB in relation to each other, or whether CBB is used at all.
+-- Used to skip unsupported modes such as unrelated inverses.
+-- TODO: Possibly some RGB24/32 stuff too?
+function BB_mt.__index:canCBB2(other)
```

Comment:
You named it canCBB**2** because it is a 2nd function and you did not find a better name?
Or because it's supposed to read/sound like "can CBB *to* (other)" ? :)
If the latter, I'd rather see it spelled canCBBto(other) or canCBBwith(other), rather than us starting writting code like SMSs :)

Reply:
Or a cozy `canCBBTogether`. ^_^

Reply:
can rename it to canCbbTo if you're getting this impression.

Reply:
Hmm, or better yet, collapse the two by making the argument optional. Not really sure why I kept it split in two in the first place.

Reply:
I think the naming/splitting is quite reasonable. At a glance I'd think `canCBB` was a property of a single one and the fact that an optional argument points to an "other" is far from self-explanatory, while it is in `canCBBTo`.

Ease of maintenance/skimming/reading > being clever to me. ;-)

Reply:
Reads indeed better after the "Cosmetics." commit, thanks.